AY2021S2-CS2103-T14-1/tp
# Docs(DG): Update command history section

## What this does
This PR:
1. fixes #415 
2. fixes #421 
2. updates the command history section of the DG to reflect updates to `CommandHistorySelector`
3. simplifies some diagrams for clarity

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the DG has the changes listed above.
